xgi/convert.py

```diff
@@ -302,3 +306,103 @@ def to_incidence_matrix(H, sparse=True, index=False):
     from_incidence_matrix
     """
     return xgi.incidence_matrix(H, sparse=sparse, index=index)
+
+
+def from_bipartite_network(G, create_using=None, dual=False):
+    """
+    Create a Hypergraph from a NetworkX bipartite graph.
+
+    Parameters
+    ----------
+    G : nx.Graph()
+        A networkx bipartite graph. Each node in the graph has a property
+        'bipartite' taking the value of 0 or 1 indicating the type of node.
```

Comment:
Following the other comments, it might be worth stressing that these are the two sets, denoted as 0 and 1, that in the hypergraph representation would be translated into hyperedges and nodes (or the other way around).

Reply:
This should now be addressed in the docstring.